pkg/search/result.go

```diff
@@ -113,6 +171,53 @@ func (repo Repository) ExportData(fields []string) map[string]interface{} {
 	return data
 }
 
+func (issue Issue) IsPullRequest() bool {
+	return issue.PullRequestLinks.URL != ""
+}
+
+func (issue Issue) ExportData(fields []string) map[string]interface{} {
+	v := reflect.ValueOf(issue)
+	data := map[string]interface{}{}
+	for _, f := range fields {
+		switch f {
+		case "assignee":
+			data[f] = map[string]interface{}{
+				"id":    issue.Assignee.ID,
+				"login": issue.Assignee.Login,
+				"type":  issue.Assignee.Type,
+			}
+		case "Labels":
+			labels := make([]interface{}, 0, len(issue.Labels))
+			for _, label := range issue.Labels {
+				labels = append(labels, map[string]interface{}{
+					"color": label.Color,
+					"id":    label.ID,
+					"name":  label.Name,
+					"url":   label.URL,
+				})
+			}
+			data[f] = labels
+		case "PullRequestLinks":
+			data[f] = map[string]interface{}{
+				"diffUrl":  issue.PullRequestLinks.DiffURL,
+				"htmlUrl":  issue.PullRequestLinks.HTMLURL,
+				"patchUrl": issue.PullRequestLinks.PatchURL,
+				"url":      issue.PullRequestLinks.URL,
+			}
+		case "User":
```

Comment:
I don't seem to be able to query the "user" field:
```
$ gh issue list --json user
Unknown JSON field: "user"
```
Furthermore, due to ambiguity of "user", I think this field should be called `author`. This will also be consistent with `gh issue list --json`.